processing-service-core/api/swagger/swagger.yaml

```diff
@@ -952,7 +954,246 @@ paths:
+          schema:
+            $ref: "#/definitions/errorMessage"
+        404:
+          description: Indicates key was not found.
+    delete:
+      tags:
+        - Tenant
+      summary: Delete tenant specific config for the provided key.
+      description: Delete tenant specific config for the provided key.
+      operationId: deleteTenantConfig
+      responses:
+        204:
+          description: Deletes the tenant specific config for the supplied key.
+        400:
+          description: Indicates invalid parameter was passed.
+          schema:
+            $ref: "#/definitions/errorMessage"
+        404:
+          description: Indicates key was not found.
+  #
+  /tenants/{tenantId}/:
+    x-swagger-router-controller: tenant
+    parameters:
+      - name: tenantId
+        in: path
+        type: string
+        description: The ID of the tenant.
+        required: true
+    put:
+      tags:
+        - Tenant
+      parameters:
+        - name: tenantConfigs
+          description: The json string representation key-value pair map of custom tenant configuration.
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/tenantConfigs"
+      summary: Creates or updates tenant specific configs.
+      description: Creates or updates a custom configuration setting for this key against the tenant.
+      operationId: setTenantConfigs
+      responses:
+        204:
+          description: Indicates that the config was created successfully.
+        400:
+          description: Indicates invalid parameter was passed.
+          schema:
+            $ref: "#/definitions/errorMessage"        
+    get:
+      tags:
+        - Tenant
+      summary: Retrieve tenant specific configs as map of key-value pairs.
+      description: Returns all tenant specific configs as a list.
+      operationId: getTenantConfigs
+      responses:
+        200:
+          description: |
+            Returns the map of tenant specific configs.
+          schema:
+            $ref: "#/definitions/tenantConfigs"
+        400:
+          description: Indicates invalid parameter was passed.
+          schema:
+            $ref: "#/definitions/errorMessage"
+        404:
+          description: Indicates that there wasn't any custom configs for the provided tenant.
+    delete:
+      tags:
+        - Tenant
+      summary: Delete tenant specific configs for the provided tenantId.
+      description: Delete tenant specific config for the provided tenantId.
+      operationId: deleteTenantConfigs
+      responses:
+        204:
+          description: Indicates tenant specific config for the provided tenantId were deleted.
+        400:
+          description: Indicates invalid parameter was passed.
+          schema:
+            $ref: "#/definitions/errorMessage"
+        404:
+          description: Indicates the tenant had no custom configs to delete.
+  #
+  /tenants/{tenantId}/effectiveconfig/{key}/:
```

Comment:
Should that extra slash at the end be there?

Reply:
No. Remove the slash from the end of the paths.